docs/configuration-parameters.md

```diff
@@ -16,152 +16,135 @@
 -->
 
 # Configuring CarbonData
- This tutorial guides you through the advanced configurations of CarbonData :
- 
+ This guide explains the configurations that can be used to tune CarbonData to achieve better performance.Some of the properties can be set dynamically and are explained in the section Dynamic Configuration In CarbonData Using SET-RESET.Most of the properties that control the internal settings have reasonable default values.They are listed along with the properties along with explanation.
```

Comment:
suggest removing this sentence : Some of the properties can be set dynamically and are explained in the section Dynamic Configuration In CarbonData Using SET-RESET